src/components/MarkdownTextArea/index.jsx

```diff
@@ -0,0 +1,128 @@
+import classNames from 'classnames';
```

Comment:
Let's make the first child of the markdown have `marginTop: 0` that way a `# Test` won't push the text down.

Reply:
To clarify, when we put # Test in the input, we'd like the <h1>Test</h1> to have a marginTop: 0 right?

I'm having trouble figuring how exactly I can apply styling to the first child of the Markdown component.

Reply:
> To clarify, when we put # Test in the input, we'd like the <h1>Test</h1> to have a marginTop: 0 right?

Right. Also, to target the first child, you can use css selectors, namely `:first-child` on the container https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child.